code/datums/autolathe/organ.dm

```diff
@@ -0,0 +1,68 @@
+/datum/design/organ
+	build_type = ORGAN_GROWER
+	materials = list(MATERIAL_BIOMATTER = 5)
+
+/datum/design/organ/ex_lungs
+	name = "Extended Lungs"
+	materials = list(MATERIAL_BIOMATTER = 30)
+	build_path = /obj/item/organ/internal/lungs/long
+
+/datum/design/organ/huge_heart
+	name = "Huge Heart"
+	materials = list(MATERIAL_BIOMATTER = 45)
+	build_path = /obj/item/organ/internal/heart/huge
+
+/datum/design/organ/big_liver
+	name = "Big Liver"
+	materials = list(MATERIAL_BIOMATTER = 30)
+	build_path = /obj/item/organ/internal/liver/big
+
+/datum/design/organ/hyper_nerves
+	name = "Hypersensitive Nerves"
+	materials = list(MATERIAL_BIOMATTER = 15, MATERIAL_GOLD = 1)
+	build_path = /obj/item/organ/internal/nerve/sensitive_nerve
+
+/datum/design/organ/super_muscle
+	name = "Super Muscle"
+	materials = list(MATERIAL_BIOMATTER = 30)
+	build_path = /obj/item/organ/internal/muscle/super_muscle
+
+/datum/design/organ/ex_blood_vessel
+	name = "Extensive Blood Vessels"
+	materials = list(MATERIAL_BIOMATTER = 30, MATERIAL_PLASTIC = 2)
+	build_path = /obj/item/organ/internal/blood_vessel/extensive
+
+/datum/design/organ/heart
+	materials = list(MATERIAL_BIOMATTER = 30)
+	build_path = /obj/item/organ/internal/heart
+
+/datum/design/organ/lungs
+	materials = list(MATERIAL_BIOMATTER = 20)
+	build_path = /obj/item/organ/internal/lungs
+
+/datum/design/organ/kidney_left
+	materials = list()
+	build_path = /obj/item/organ/internal/kidney/left
+
+/datum/design/organ/kidney_right
+	materials = list()
```

Comment:
Are these meant not to have any materials in?

Reply:
That just means there is no additional material added to the cost. In this case, the kidneys will cost 20 biomatter since that's what is defined in the organ object.